README.md

```diff
@@ -1,5 +1,13 @@
 # bonjour-reflector
 
+## About this project
+
+This projects aims to make bonjour devices (such as printers, chromecasts, ...) discoverable and usable by other devices located in different VLANs.
+
+This is done by intercepting mDNS packets and forwarding them to the correct VLANs (for the request packets, the VLANs where the bonjour devices are located; for the response packets, the VLANs where the searching devices are located).
```

Comment:
Let's split this in bullet points to make it clearer maybe ?

Reply:
Should we also add a sentence about how this interception is done ? (the bonjour-reflector should have on interface with all the concerned vlans on **tagged** for example)